toolbox_runner/run.py

```diff
@@ -8,7 +8,7 @@
 try:
     stream = os.popen("docker version --format '{{.Server.Version}}'")
     DOCKER = stream.read()
-    if DOCKER == '':
+    if DOCKER == '\n':
```

Comment:
ha. maybe that's why it is not working properly on every system. (on debian, the old code used to work, I think)

Could you change the if to check for one or the other? Both are considered to having no docker available

Reply:
Yes, I just changed run.py